src/middleware/router.coffee

```diff
@@ -151,6 +161,11 @@ sendRequestToRoutes = (ctx, routes, next) ->
               # handle mediator reponse
               responseObj = JSON.parse response.body
               ctx.mediatorResponse = responseObj
+
+              if responseObj.error?
+                ctx.autoRetry = true
```

Comment:
So this is saying that the transaction is available to be auto retried - it gives us a handle to quickly query for those transactions that need to be rerun, rather than trying to sift through all failed transactions for a channel